src/index.js

```diff
@@ -11,14 +11,20 @@ export function configure() {
   const attached = Controller.prototype.attached;
   const detached = Controller.prototype.detached;
 
+  key.filter = () => true; // disable Keymaster's built-in filter
+
   Controller.prototype.attached = function() {
     // attached() gets called twice but only does things when isAttached is false
     let isAttached = this.isAttached;
     attached.call(this);
 
     if (!isAttached) {
       eachMethod(this.viewModel, value => {
-        key(value.combo, e => {
+        key(value.combo.shortcuts, e => {
+          if (!value.combo.runInsideInputs && !defaultKeymasterFilter(e)) {
+            return true;
```

Comment:
This can be just `return;` I think.
The return value here is not used by aurelia.